src/constants.ts

```diff
@@ -62,6 +62,9 @@ export const DEFAULT_ADVANCED_SETTINGS = {
     [AdvancedCommandAction.SONG_START_DELAY]: 3,
 };
 
+export const CLIP_REPLAY_DELAY = 1000;
```

Comment:
Are these used anywhere other than session?

Reply:
That's right. I planned on introducing an `/advanced` option to configure this so I preemptively moved it there